### PR TITLE
Add information to install libffi-dev, needed for python-mirobo

### PR DESCRIPTION
**Pull request by olskar**
**Description:**
libffi-dev is needed for home assistant to install python-mirobo.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8938

### CODE REVIEW

File: source/_components/vacuum.xiaomi.markdown [hunk 1]
```diff
@@ -20,6 +20,11 @@ Current supported features are `turn_on`, `pause`, `stop`, `return_to_home`, `tu
 
 Follow the pairing process using your phone and Mi-Home app. From here you will be able to retrieve the token from a SQLite file inside your phone.
 
+Before you begin you need to install libffi-dev by running the command below. This is needed for python-mirobi to be installed correctly. 
+```bash
+$ apt-get install libffi-dev
```
Comment (azogue) [inline source/_components/vacuum.xiaomi.markdown]: remove the `$ ` and it's good to go

Comment (azogue) [inline source/_components/vacuum.xiaomi.markdown]: and if you encapsulate `libffi-dev` and `python-mirobi` to proper formatting, it would be perfect

**Comment by azogue** [comment]
👍

**Comment by sebdoan** [comment]
@syssi Thanks for updating the information on the component page, however I can't understand the instructions or for some reason they don't work for me. I'm running HASSBIAN so my HASS instance is running VirtualEnv. 

When logged into my Raspbery Pi:
1. I run "**sudo apt-get install libffi-dev**" and it all installs fine
2. Then i run "**sudo su -s /bin/bash homeassistant**" then "**source /srv/homeassistant/bin/activate**", this all works fine and gets me into the VirtualEnv
3. I assume what you mean by step 2 to to actually install libffi-dev in the VirtualEnv, but no combination of "**pip install libffi-dev**" or "**apt-get install libffi-dev**" actually works

And i still get the error as below. HELP???

_[homeassistant.util.package] Unable to install package python-mirobo==0.1.2_

**Comment by syssi** [comment]
There is another dependency which must be fulfilled (libssl-dev):

https://community.home-assistant.io/t/issue-config-xiaomi-vacuum-by-v0-51-1/24304/7

**Pull request merged**